tools-v2/pkg/cli/command/curvebs/check/copyset/status.go

```diff
@@ -0,0 +1,103 @@
+/*
```

Comment:
Can this be packaged into a command like [chunkserver](https://github.com/opencurve/curve/blob/44b695e7ed9f83144f19712e804dc56899ce2e5d/tools-v2/pkg/cli/command/curvebs/query/chunk/chunkserver.go) ?Maybe this command is only called by other commands and does not need to be added. Once it is added, it is convenient to modify.